manim/utils/cfgwriter.py

```diff
@@ -58,7 +63,7 @@ def main():
         console.print("Enter the Style for %s" % key + ":", style=key, end="")
         temp = input()
         if temp:
-            while check_valid_colour(temp) == False:
+            while check_valid_colour(temp):
```

Comment:
This line should be:
```suggestion
            while not check_valid_colour(temp):
```
The current while loop will run as long as the colour is valid. The suggested loop should run if the colour is invalid, which is the intended behaviour.